server/mergin/sync/private_api_controller.py

```diff
@@ -69,7 +69,7 @@ def create_project_access_request(namespace, project_name):  # noqa: E501
             "html": render_template(
                 "email/project_access_request.html",
                 expire=access_request.expire,
-                link=f"{request.url_root.rstrip('/')}/projects/{namespace}/{project.name}/settings",
+                link=f"{request.url_root}",
```

Comment:
This would lead to current selected workspace of owner not a project workspace, no?

Reply:
yes, currently there is no way how to pass workspace in url for dashboard page, there is probably same issue for other email links

there is an issue for this: https://github.com/MerginMaps/server-private/issues/2221

Reply:
Just change settings -> collaborators page. There is new place for access requests for admins. We do not want to redirect to dashboard. ?workspace_id is working just for subscription page.

Reply:
https://app.dev.merginmaps.com/projects/problem/test/collaborators